### DISCUSSION TITLE
How do i extract annotations and highlights from a markded up pdf into .txt / .md format

### DISCUSSION THREAD

**Discussion by Grover3293** [BryanJenksCommunity/FAQ#11] [Q&A] [answered]
there is a site https://www.sumnotes.net/ but it is a type of freemium. so i can't extract annotations after certain limit. 
I also found a software on github [[DYextractor]] which works on java but its not running/recognising the file contained within 
Please suggest some program for this.

**Comment by tallguyjenks** [comment]
@Grover3293 are the pdf's in zotero?

**Comment by Grover3293** [comment]
No. These are in my local explorer only.
Can zotero show highlights/annotations even if those aren't made in the software apriori?

**Comment by tallguyjenks** [comment]
@Grover3293 i believe so. Once the highlights and annotations are on the pdf, input into zotero and you could follow the [MD notes](https://forum.obsidian.md/t/zotero-best-practices/164/57) workflow for extraction to markdown

**Comment by Grover3293** [comment]
Relieved to find it worked ☺️☺️

**Comment by Grover3293** [comment]
My bad! You have already explained it in detail in your comprehensive overview video.
Phew...
Sorry for issue 😅

**Comment by tallguyjenks** [comment]
@Grover3293 No worries :)

**Comment by mmarco2121** [comment]
@tallguyjenks Thanks for the fantastic video and your leadership in the Obsidian forums. I tried following along with your video. I installed the plugins and successfully marked up a PDF. I see the notes in Zotero. When I try to export to markdown I get the pop up box asking me to choose a folder. Once I select a folder nothing happens. No markdown files are created. Not sure what to do. I am running on Windows 10, latest updates for everything.

**Comment by tallguyjenks** [comment]
@mmarco2121 You're welcome! I'm glad you enjoy it! Can you post some screen shots of the issue? namely the pdf itself, the extracted annotations, the markdown (if it work in zotero at least) file in zotero, the folder you selected for your markdown document download, etc :) this will help with troubleshooting!

**Comment by mmarco2121** [comment]
@tallguyjenks I used this site with the Zotero plugin: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7137172/

Marked up the PDF as per your video. Results below.

![image](https://user-images.githubusercontent.com/10320927/100167928-35eeac80-2e8e-11eb-9b06-4e6e9e50ba17.png)

When I right click the notes and Export as Markdown (or any of the MD options really), I get this pop up:

![image](https://user-images.githubusercontent.com/10320927/100168017-6cc4c280-2e8e-11eb-8b6b-3736c3ebcb08.png)

When I select my folder, in this case downloads, the window closes and nothing happens.

Plugins installed: 

![image](https://user-images.githubusercontent.com/10320927/100168068-87973700-2e8e-11eb-9825-d02c2e5c9d72.png)

**Comment by tallguyjenks** [comment]
@mmarco2121  hmm.. everything should be working correctly. Just for the heck of it i tried to re-export one of my files and i too was unable to get any markdown output.

i first
- update all plugins
- completely kill and restart zotero
- right click on the extracted annotations file and try each option, Export markdown, batch export
    - if none of those work, then you may need to file an issue on the [MDNotes repo with Argentum](https://github.com/argenos/zotero-mdnotes)

let me know if any of these resolve the issue. this is not normal behavior 🤔

**Comment by mmarco2121** [comment]
@tallguyjenks thanks. Still no success. I'll reach out to the plug-in creator.

**Comment by Grover3293** [comment]
faced the same issue but resolved by :- first create the annotations --> right click on annotations and export to markdown - a md file will be created alongside extracted annotations but not on folder you chose. you just drag  the file directly on the concerned folder from zotero only.
hope this workaround makes sense😊😊

**Comment by mmarco2121** [comment]
@Grover3293 I don't think I follow. I successfully extracted the annotations from the PDF and have them in Zotero. When I right click the extracted annotations and select Export to Markdown, or Create MDnotes file or any other option, I get a pop up box asking me to select the folder. When I select the folder, the Window disappears and nothing happens. No files are created.

**Comment by tallguyjenks** [comment]
@Grover3293 What OS do you use? Windows, Mac, Linux?

**Comment by tallguyjenks** [comment, marked as answer]
Converted from issue
